frontend/src/components/ResizeTable/ResizeTable.tsx

```diff
@@ -63,8 +64,10 @@ function ResizeTable({
 			...restProps,
 			components: { header: { cell: ResizableHeader } },
 			columns: mergedColumns,
+			hideOnSinglePage: true,
```

Comment:
we need `hideOnSinglePage` both at this level and the pagination level ?

Reply:
No, we just need it only on pagination level. I had added it here by mistake :sweat_smile: . will discard this change :slightly_smiling_face: